smarts/sstudio/genhistories.py

```diff
@@ -26,15 +26,22 @@
 import sqlite3
 import struct
 import sys
-
 import ijson
 import numpy as np
 import pandas as pd
 import yaml
 from numpy.lib.stride_tricks import as_strided as stride
-from waymo_open_dataset.protos import scenario_pb2
 from typing import Dict, Generator, Union
 
+try:
+    from waymo_open_dataset.protos import scenario_pb2
+except ImportError:
+    print(sys.exc_info())
+    print(
+        "You may not have installed the [waymo] dependencies required to use the waymo replay simulation. Install them first using the command `pip install -e .[waymo]` at the source directory."
+    )
+    pass
```

Comment:
Printing `sys.exc_info()` directly, without re-formatting it according to our needs, will appear a little messy. `pass` may not be needed at the end.
 
```suggestion
try:
    from waymo_open_dataset.protos import scenario_pb2
except ImportError:
    print(
        "You may not have installed the [waymo] dependencies required to use the waymo replay simulation. Install them first using the command `pip install -e .[waymo]` at the source directory."
    )
```